scripts/generate_pull_request_pipelineruns.py

```
@@ -0,0 +1,237 @@
+    cel_key = 'pipelinesascode.tekton.dev/on-cel-expression'
+    original_on_cel_expression = get_exact_formatted_value(push_pipeline_lines, annotations.lc.key(cel_key),
+                                                           annotations.lc.value(cel_key))
+
+    pr_on_cel_expression = (original_on_cel_expression
+                            .replace('"push"', '"pull_request"')
+                            .replace('-push.yaml"', '-pull-request.yaml"')
+                            + '&& body.repository.full_name == "opendatahub-io/notebooks"')
+
+    component = push_pipeline['metadata']['labels']['appstudio.openshift.io/component']
+
+    build_platforms = ['linux/x86_64']
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)